test-requirements.txt

```diff
@@ -29,3 +29,4 @@ testtools>=1.4.0
 mock
 tabulate>=0.7.5
 voluptuous!=0.13.0
+SQLAlchemy==1.4.47
```

Comment:
Since we use AIM in different distros (Ubuntu 18.04, 20.04, RHEL7, RHEL8), I don't think we should specify the constraint this way. Instead, we may want to limit it to be below a certain version. That may take experimentation. We also should probably do that as a separate patch.